js/controller.js

```diff
@@ -78,7 +78,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "mainVideoPlayhead": 0,
       "adVideoPlayhead": 0,
       "focusedElement": null,
-      "playPauseButtonFocused": false,
+      "focusedControl": null, // Stores the id of the control bar element that is currently focused
```

Comment:
Is it possible for multiple controls to be focused somehow?

Reply:
Nope, only one at a time, same as with `document.activeElement`